### ISSUE TITLE
Change bg color of image areas

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#149]
### Location
http://test-dukes.pantheonsite.io/news/
http://test-dukes.pantheonsite.io/stories/
http://test-dukes.pantheonsite.io/stories/history/

### Description
Please change the bg color of the Free Jar Friday card and the Stories feature image to black for conformance (just like we did with the recipe cards).

<img width="507" alt="screen shot 2016-12-14 at 3 06 20 pm" src="https://cloud.githubusercontent.com/assets/1574420/21198867/0c4a7fde-c20f-11e6-92b4-bd13a9c302eb.png">

**Comment by caitlynmayers** [comment]
@hessdev i am still seeing this issue on the Stories and History pages

**Comment by caitlynmayers** [comment]
So the image bg provides the adequate contrast, but the element itself doesn't have the bg applied that passes the checkers.

**Comment by caitlynmayers** [comment]
Fix verified.
